### PR TITLE
[main][SPECS-EXTENDED] Fixing `bluez`, `dyninst`, `exempi`, `fakechroot`, `gstreamer1`, `libproxy`.

### PR DESCRIPTION
**Pull request by PawelWMS**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [X] The toolchain has been rebuilt successfully (or no changes were made to it)
- [X] The toolchain/worker package manifests are up-to-date
- [X] Any updated packages successfully build (or no packages were changed)
- [X] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [X] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [X] All package sources are available
- [X] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [X] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [X] All source files have up-to-date hashes in the `*.signatures.json` files
- [X] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [X] Documentation has been updated to match any changes to the build system
- [X] Ready to merge

---

###### Summary 

Another batch of `SPECS-EXTENDED` fixes.

###### Change Log  

- Fixed: `bluez`, `dyninst`, `exempi`, `fakechroot`, `gstreamer1`, and `libproxy`.
- Removed: `advancecomp`, `clutter*`, `cogl`, `cpptest`, `glog`, and `libchamplain`.
- Removed accidentally added `*.orig` file (#2387).

###### Does this affect the toolchain?  

No.

###### Associated issues  

- #2387

###### Test Methodology

- Local package builds.

### CODE REVIEW

File: SPECS-EXTENDED/gstreamer1/gstreamer1.spec [hunk 1]
```diff
@@ -72,51 +55,30 @@ Conflicts:      gstreamer1-plugins-bad-free-devel < 1.13
...
-  --disable-fatal-warnings \
-  --disable-silent-rules \
-  --disable-tests --disable-examples \
-  --with-ptp-helper-permissions=capabilities
-
-
-%make_build V=1
-
+%meson	\
+  -D package-name='Fedora GStreamer package' \
```
Comment (jslobodzian) [inline SPECS-EXTENDED/gstreamer1/gstreamer1.spec:64]: do we want to say fedora here?

File: SPECS-EXTENDED/gstreamer1/gstreamer1.spec [hunk 2]
```diff
@@ -72,51 +55,30 @@ Conflicts:      gstreamer1-plugins-bad-free-devel < 1.13
...
+  -D package-origin='http://download.fedoraproject.org' \
+  -D tests=disabled -D examples=disabled \
```
Comment (jslobodzian) [inline SPECS-EXTENDED/gstreamer1/gstreamer1.spec:66]: consider -D examples on a separate line.

should a %check build run with -D tests=enabled?

**Review by jslobodzian — APPROVED** [review]

File: SPECS-EXTENDED/gstreamer1/gstreamer1.spec (hunk 1 above)
Comment (PawelWMS) [reply SPECS-EXTENDED/gstreamer1/gstreamer1.spec:64]: I guess we can change that while we're here, you're right.

File: SPECS-EXTENDED/gstreamer1/gstreamer1.spec (hunk 2 above)
Comment (PawelWMS) [reply SPECS-EXTENDED/gstreamer1/gstreamer1.spec:66]: Honestly, I wasn't focusing on fixing tests right now, just making the package build. I'll move `-D examples...` to a separate line, though.

**Pull request merged**